app/models/project.rb

```diff
@@ -28,11 +28,19 @@ class Project < ApplicationRecord
   has_one :grade, dependent: :destroy
   has_one :project_datum, dependent: :destroy
 
+  scope :private_projects, -> { where(project_access_type: "Private") }
```

Comment:
This will return private projects in search. Is that intended?

Reply:
Oh sorry, I need to pass current_user as a parameter to scope. I will do that in the next commit.

Reply:
@gr455 Kindly review the commit.